cli/components/list_raw_object.go

```diff
@@ -0,0 +1,65 @@
+// Copyright (c) Mondoo, Inc.
+// SPDX-License-Identifier: BUSL-1.1
+
+package components
+
+import (
+	"fmt"
+	"strings"
+	"text/tabwriter"
+
+	"github.com/rs/zerolog/log"
+	"go.mondoo.com/cnquery/v11/cli/theme"
+)
+
+// Object is the interface that a list need to implement so we can display its objects.
+type Object interface {
```

Comment:
can we have a more specific name for this interface? I find this one too generic making it unclear what it's purpose is. Perhaps call it `CliListableItem` or something similar?